packages/graphiql/cypress/support/commands.js

```diff
@@ -12,6 +12,33 @@
 Cypress.Commands.add('getCy', cyName => {
   return cy.get(`[data-cy=${cyName}]`);
 });
+
+Cypress.Commands.add('clickExecuteQuery', () => {
+  return cy.get('.execute-button').click();
+});
+
+Cypress.Commands.add('clickPrettify', () => {
+  return cy.get('[title="Prettify Query (Shift-Ctrl-P)"]').click();
+});
+
+Cypress.Commands.add('visitWithOp', ({ query, variables }) => {
+  let url = `/?query=${encodeURIComponent(query)}`;
+  if (variables) {
+    url += `&variables=${encodeURIComponent(variables)}`;
+  }
+  return cy.visit(url);
+});
+
+Cypress.Commands.add('assertQueryResult', (op, mockSuccess) => {
+  cy.visitWithOp(op);
+  cy.clickExecuteQuery();
+  cy.wait(500);
```

Comment:
If this is necessary, the test is probably flaky. We should fix this in a follow up PR; for now:

```suggestion

  // TODO: we should re-write this test so the wait isn't necessary
  cy.wait(500);

```

Though I guess it also comes up under the Cypress ESLint checks? :grin:

Reply:
the only way to get around this is probably to mock the schema entirely client side. @tgriesser has some utilities, as well as some patterns I've found to work as well, so it's something we will have to address more properly down the road :(